src/exact.jl

```diff
@@ -68,26 +68,34 @@ function isdominated(A, b, islb, isfun, k, solver, lb, ub, epsilon)
         if ix != k
             ic += 1
             δb = b[k] - b[ix]
-            @inbounds h[ic] = ((isfun&islb) || (~isfun&~islb)) ? δb : -δb
+            @inbounds h[ic] = ((isfun & islb) || (~isfun & ~islb)) ? δb : -δb
             @inbounds H[ic, 1] = (islb) ? 1 : -1
 
             for jx in 1:nx
                 dl = A[ix, jx] - λk[jx]
-                @inbounds H[ic, jx+1] = (islb) ? dl : -dl
+                @inbounds H[ic, jx + 1] = (islb) ? dl : -dl
             end
         end
     end
 
     # update lower and upper bound if Vector
     lbx = isa(lb, Vector) ? vcat(-Inf, lb) : lb
-    ubx = isa(ub, Vector) ? vcat( Inf, ub) : ub
-
-    # solve the LP with MathProgBase
-    res = linprog(c, H, -Inf, h, lbx, ubx, solver)
-    if res.status == :Optimal
-        res = res.objval
-        return (islb) ? -res < epsilon : res > -epsilon
-    else
-        return false
+    ubx = isa(ub, Vector) ? vcat(Inf, ub) : ub
+
+    # solve the LP with JuMP
+    model = Model(optimizer_constructor)
```

Comment:
I understand the need to use `JuMP` here, even if it is a non-trivial dependency. In the long term, I would suggest to have a look on a MOI `linprog` , e.g. as in [this package](https://github.com/joaquimg/MatrixOptInterface.jl).

Reply:
I had a brief look at MatrixOptInterface.jl but wasn't sure if it was ready and/or maintained.
Is there another simple replacement for MathProgBase.linprog?

Reply:
It's still premature to use MatrixOptInterface. Using MOI directly could have the advantage of not being tied to `Float64`. Jump is ok for now